addons/website_event/tests/test_event_internals.py

```diff
@@ -30,18 +46,10 @@ def test_menu_management_frontend(self):
             'website_menu': True,
             'community_menu': False,
         })
-        self.assertTrue(event.website_menu)
-        self._assert_website_menus(event)
+        self._assert_website_menus(event, ['Introduction', 'Location', 'Register'], menus_out=['Community'])
 
         introduction_menu = event.menu_id.child_id.filtered(lambda menu: menu.name == 'Introduction')
         introduction_menu.unlink()
 
         self.assertTrue(event.website_menu)
-        self._assert_website_menus(event, set(['Location', 'Register']))
-
-    @users('user_eventmanager')
-    def test_menu_update(self):
```

Comment:
Not testing the update anymore?
I see you test create twice in the base test, is it sufficient?

Reply:
Added an update on website_menu to False on a test to ensure it is also tested, right.